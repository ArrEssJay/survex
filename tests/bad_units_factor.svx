; pos=fail error=6
*fix 1 reference 0 0 0
*units tape 0 feet
*units compass 000 degrees
*units clino 0.0 grads
*units depth .0 metres
*units altitude 0 yards
*units backclino 0 percent
