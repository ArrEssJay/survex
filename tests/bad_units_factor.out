./bad_units_factor.svx:3:14: *UNITS factor must be non-zero
./bad_units_factor.svx:4:17: *UNITS factor must be non-zero
./bad_units_factor.svx:5:15: *UNITS factor must be non-zero
./bad_units_factor.svx:6:15: *UNITS factor must be non-zero
./bad_units_factor.svx:7:18: *UNITS factor must be non-zero
./bad_units_factor.svx:8:19: *UNITS factor must be non-zero

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 1 survey station, joined by 0 legs.
There are 0 loops.
Total length of survey legs =    0.00m (   0.00m adjusted)
Total plan length of survey legs =    0.00m
Total vertical length of survey legs =    0.00m
   1 0-node.
There were 0 warning(s) and 6 error(s) - no output files produced.
